packages/grid/_modules_/grid/models/api/gridLocaleTextApi.ts

```diff
@@ -1,7 +1,12 @@
+import { Localization as CoreLocalization } from '@material-ui/core/locale';
+
+// @ts-ignore
```

Comment:
It might be better to use 
```suggestion
// @ts-expected-error THE REASON
```

Reply:
Done